backend/kitschy_api/urls.py

```diff
@@ -1,14 +1,15 @@
 from django.urls import include, path
-from kitschy_api.views.address import AddressViewSet
+from kitschy_api.views import AddressViewSet
```

Comment:
Inconsistent import, should be `from kitschy_api.views.address import AddressViewSet`.